test/system/271-tcp-cors-server.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests podman system service under systemd socket activation
+#
+
+load helpers
+load helpers.systemd
+
+SERVICE_NAME="podman_test_$(random_string)"
+
+SERVICE_TCP_ADDR="localhost:3113"
+
+SERVICE_FILE="$UNIT_DIR/$SERVICE_NAME.service"
+SOCKET_FILE="$UNIT_DIR/$SERVICE_NAME.socket"
+
+@test "podman system service - tcp CORS " {
+    skip_if_remote "system service tests are meaningless over remote"
+    run_podman system service --cors=* tcp:$SERVICE_TCP_ADDR -t 20 &
+    podman_pid="$!"
+    sleep 5s
```

Comment:
`sleep` is a bad idea, but I'll allow it for now because `wait_for_port` is broken (emergency PR in the works)